platform/core/src/services/HangingProtocolService/HangingProtocolService.js

```diff
@@ -314,7 +342,12 @@ class HangingProtocolService {
 
       highestStudyMatchingScore = studyMatchDetails.score;
 
+      // TODO - use display sets instead of series, as that will use the
+      // correct re-grouped information.
       study.series.forEach(aSeries => {
+        if (existingMatches && existingMatches.indexOf(aSeries) != -1) {
```

Comment:
Allows matching consecutive series, without re-displaying the same one multiple times.  Need to figure out a good way to allow re-matching an existing instance.  I was thinking of adding viewport rules that will find a previous series again.

Reply:
what if it is fusion? pt-axial, ct-axial, fusion-axial (both pt-axial, ct-axial)

Reply:
we are actually moving to `displaySets` in future (because of fusion stuff), so I would suggest to stick with the series matching by rules only